task_5/Tatiana.Derevenets/src/main/java/task_5/HappyTicket.java

```diff
@@ -0,0 +1,25 @@
+package task_5;
+
+import java.util.Scanner;
+
+public class HappyTicket {

```

Comment:
Rewrite the task the same way you did in the class. Do not just copy, write it by yourself from scratch